packages/gatsby/cache-dir/loader.js

```diff
@@ -66,7 +66,7 @@ const cleanAndFindPath = rawPathname => {
     return cleanAndFindPathCache[trimmedPathname]
   }
 
-  let foundPath = findMatchPath(matchPaths, trimmedPathname)
+  let foundPath = findMatchPath(trimmedPathname)
```

Comment:
Leaving this for anyone reviewing after me:

This function is being changed to not accept `matchPaths` because, `loader.addMatchPaths(matchPaths)` is typically called before it which inits the matchPaths inside loader

Reply:
Not typically, always 😄